pkgs/applications/kde/marble.nix

```diff
@@ -18,6 +18,11 @@ mkDerivation {
     qtscript qtsvg qtquickcontrols qtwebengine shared-mime-info krunner kparts
     knewstuff gpsd
   ];
+  # Remove once upstream supports gpsd 3.23.1
```

Comment:
Please add a link to the upstream MR @CrystalGamma mentioned in #139794 (https://invent.kde.org/education/marble/-/merge_requests/57). I would prefer to use the patch from that MR directly, but if you prefer `substituteInPlace` that is also okay.